src/Compute/Compute/Manual/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -285,6 +293,8 @@ public async Task<ResourceConfig<VirtualMachineScaleSet>> CreateConfigAsync()
 
                 var proximityPlacementGroup = resourceGroup.CreateProximityPlacementGroupSubResourceFunc(_cmdlet.ProximityPlacementGroupId);
 
+                var hostGroup = resourceGroup.CreateDedicatedHostGroupSubResourceFunc(_cmdlet.HostGroupId);
```

Comment:
The process here to set the hostGroup value is different than in src/Compute/Compute/Strategies/ComputeRp/VirtualMachineStrategy.cs . 
Is that expected? Should they be the same? 
Which is better? This method seems like extra work.

Reply:
Yeah, I can see why it might be unnecessary work. I like this method because it relies more on an error from CRP than the Subresource class not being able to parse a hostgroup resource id. I am not sure which way is advised. I am fine with having this removed, but I am not sure if it's the best move. Plus, this way matches the way the other subresource (`proximityPlacementGroup` here) is set

Reply:
And I was matching another Subresource's pattern in VMStrategy. 
Adding more checks and error catching is fine by me, so I think leaving it as is is fine. And keeping the different files consistent is good. 
No changes needed.